cime_config/acme/config_files.xml

```diff
@@ -32,6 +32,14 @@
     <desc>file containing specification of all supported model grids, domains and mapping files (for documentation only - DO NOT EDIT)</desc>
   </entry>
 
+  <entry id="CONFIG_TESTS_FILE">
+    <type>char</type>
+    <default_value>$CIMEROOT/scripts/Testing/Testcases/config_tests.xml</default_value>
```

Comment:
Where to find the test descriptions.   We may want to move this into the cime_config/$model directory at some point.